Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/service/impl/EventLogFileServiceImpl.java

```diff
@@ -34,41 +36,54 @@
 import org.springframework.stereotype.Service;
 
 @Service
-public class EventLogFileServiceImpl implements EventLogFileService  {
+public class EventLogFileServiceImpl implements EventLogFileService {
     private String directoryPath;
-    
+
     @Inject
     public EventLogFileServiceImpl(final ConfigBean configBean) {
-        this.directoryPath = configBean.getLogsDir();
-    }
-    
-    @Override
-    public void serializeXLog(XLog log, String name, XSerializer serializer) throws Exception {
-        FileOutputStream outputStream;
-        try {
-            File directory = new File(directoryPath);
-            if (!directory.exists()) directory.mkdirs();
-            File file = new File(directoryPath + name);
-            if (!file.exists()) file.createNewFile();
-            outputStream = new FileOutputStream(file);
-            serializer.serialize(log, outputStream);
-            outputStream.close();
-        } catch (Exception e) {
-            e.printStackTrace();
-            System.out.println("Error");
-        }
+	this.directoryPath = configBean.getLogsDir();
     }
-    
+
+//    @Override
+//    public void serializeXLog(XLog log, String name, XSerializer serializer) throws Exception {
+//        FileOutputStream outputStream;
+//        try {
+//            File directory = new File(directoryPath);
+//            if (!directory.exists()) directory.mkdirs();
+//            File file = new File(directoryPath + name);
+//            if (!file.exists()) file.createNewFile();
+//            outputStream = new FileOutputStream(file);
+//            serializer.serialize(log, outputStream);
+//            outputStream.close();
+//        } catch (Exception e) {
+//            e.printStackTrace();
+//            System.out.println("Error");
+//        }
+//    }
```

Comment:
What's the reasoning for holding on to this commented-out block?

Reply:
I will delete it.